### PR TITLE
SEO-174654-Angular-description-too-short-long

### PR DESCRIPTION
**Pull request by Omanyadick**
Hi @Aishwarya-Ganesan, Kindly review my PR below. and additionally, I have worked on image alt text due to CI issue
Title | Description
-- | --
|Task Category | Site audit Multiple description to short|
|Content Task Link/Mail Screenshot | NA|
|Hotfix | https://github.com/syncfusion-content/angular-docs/pull/380|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/188676/|
|Work link | https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7BCCB6349C-5878-4E5E-A945-29E1035BB011%7D&file=Dickson.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change | Reason for not identifying earlier
-- | -- | --
 Multiple description to short | Improving Crawl Efficiency and also it Ensures Proper Indexing. | This is because of delayed Crawling and Indexing|

Regards
Dickson J

### CODE REVIEW

File: angular/Barcode/Getting-Started.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Getting Started for Angular Barcode
-description: Barcode
-platform: Angular
+title: Getting Started with Angular Barcode Component | Syncfusion
+description: Checkout and learn about Getting started with Angular Barcode component of Syncfusion Essential JS 2 and more details.
```
Comment (Yvone-Atieno) [inline angular/Barcode/Getting-Started.md:3]: Learn here all about getting started with Syncfusion Essential Angular Barcode control, its elements, and more details.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angular/Barcode/Getting-Started.md (hunk 1 above)
Comment (Omanyadick) [reply angular/Barcode/Getting-Started.md]: Updated as proposed

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**